src/components/Dropdown/index.js

```diff
@@ -22,12 +22,14 @@ const Dropdown = ({
   label,
   handleChange,
   items,
+  error,
   disabled = false,
 }) => (
   <Box sx={{ minWidth: 120, color: '#4F5F30' }}>
     <FormControl fullWidth>
       <InputLabel>{label}</InputLabel>
       <Select
+        sx={error && { '.MuiOutlinedInput-notchedOutline': { borderColor: 'rgba(255, 0, 0, .5)' } }}
```

Comment:
maybe change it to `sx={{ '.MuiOutlinedInput-notchedOutline': error && { borderColor: 'rgba(255, 0, 0, .5)' } }}` to prevent warning in console

Reply:
also update the `borderColor` for this or for the NumberTextField to make them consistent on SiteCondition form

Reply:
I set the prop name to emptyWarning to be a little more clear on what its doing.  Its the same for both the text box and the drop down and can now be implemented anywhere in the application by adding the prop emptyWarning and setting a Boolean to true when you want the warning to show.